### DISCUSSION TITLE
Un filtering previously filtered out hints or errors

### DISCUSSION THREAD

**Discussion by asafm7** [apache/netbeans#6062] [Q&A] [answered]
Once hints or errors are filtered out, how can it be undone? Where can filtering rules be managed?

More specifically in the case of filtering out only for a specific file.

**Comment by asbachb** [comment]
On NetBeans Level: `Tools -> Options -> Editor -> Hints`
On Project Level: `Right Click your project -> Properties -> Hints`
On File Level: Don't know.

**Comment by asafm7** [comment]
Thanks @asbachb On File Level is the piece I'm missing.

**Reply by asbachb** [reply]
Which file type / file extension we're talking about?

**Comment by asafm7** [comment]
@asbachb I think it is the same for all, but for example PHP

**Comment by asafm7** [comment, marked as answer]
In case someone else is looking for it, I found it.

After filtering out a hint, it appears in a list at the top of the page. Clicking on it in the list re-enables it.
